app/lib/bk/compat/parsers/bitbucket/variables.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-require_relative '../../pipeline/step'
-require_relative '../../pipeline/plugin'
+require_relative '../../models/plugin'
+require_relative '../../models/steps/input'
```

Comment:
This file does not appear to be using plugins so that dependency is not necessary:

```suggestion
require_relative '../../models/steps/input'
```

Reply:
Adjusted 👍